tools/ci-build/changelogger/src/main.rs

```diff
@@ -5,37 +5,49 @@
 
 use anyhow::Result;
 use changelogger::init::subcommand_init;
+use changelogger::new_entry::subcommand_new_entry;
+use changelogger::preview_next::subcommand_preview_next;
 use changelogger::render::subcommand_render;
 use changelogger::split::subcommand_split;
 use clap::Parser;
 
 #[derive(Parser, Debug, Eq, PartialEq)]
 #[clap(name = "changelogger", author, version, about)]
 pub enum Args {
-    /// Split SDK changelog entries into a separate file
-    Split(changelogger::split::SplitArgs),
+    /// Print to stdout the empty "next" CHANGELOG template
+    Init(changelogger::init::InitArgs),
+    /// Create a new changelog entry Markdown file in the `smithy-rs/.changelog` directory
+    NewEntry(changelogger::new_entry::NewEntryArgs),
+    /// Render a preview of changelog entries since the last release
+    PreviewNext(changelogger::preview_next::PreviewNextArgs),
```

Comment:
Might consider shortening these to just `new` and `ls` for creating a new entry and listing/previewing any current entries.

Reply:
Replaced `NewEntry` with `New` and `PreviewNext` with `Ls` in 1ae8781.